src/app/features/debriefings/components/molecules/DebriefForm/scaffold.tsx

```diff
@@ -1,19 +1,42 @@
+import React from "react"
 import { FormPositioner } from "amsterdam-scaffold-form/package"
 import { Fields } from "app/features/shared/components/molecules/Form/ScaffoldFields"
 import { navigate } from "@reach/router"
+import InfoButton from "app/features/shared/components/molecules/InfoHeading/InfoButton"
 
-export default (caseId: Components.Schemas.Case["id"], isEditing = false) => {
+const Scaffold = (caseId: Components.Schemas.Case["id"], isEditing = false) => {
+  const helpTextViolation = <>
+    <ol>
+      <li><b>Nader onderzoek nodig</b>
+        <ul>
+          <li>Iets uitzoeken wat niet op de locatie zelf is. Bijv. advies van de teamleider of jurist.</li>
+          <li>Verwerk de uitkomst vervolgens in een nieuwe debriefnotitie.</li>
+        </ul>
+        <br/><br/>
+      </li>
+      <li><b>Aanvullend huisbezoek nodig</b>
+        <ul>
+          <li>Nader onderzoek nodig op het adres zelf door de toezichthouders.</li>
+          <li>Zet dit bezoek uit via de projectmedewerker.</li>
+          <li>Vermeldt waar specifiek op gelet moet worden.</li>
+          <br/><br/>
```

Comment:
Would add CSS `margin-bottom` to the `<ul>` instead of (mis)using `<br/>` for styling

Reply:
Done